pip_api/_parse_requirements.py

```diff
@@ -204,7 +204,7 @@ def parse_requirements(filename, options=None, include_invalid=False):
 
             elif known.requirements:
                 if known.requirements not in parsed:
-                    to_parse.add(known.requirements)
+                    to_parse.add(os.path.join(os.path.dirname(filename), known.requirements))
```

Comment:
This line now contains 94 characters, and is the longest line in the file.  Is that too long?  I'll split it up if so.